lesson22/app.js

```diff
@@ -1,225 +1,235 @@
-const url = "https://jsondata.okiba.me/v1/json/tN7Ay210317141032";
 const parent = document.getElementById("js-parent");
-const table = document.createElement("table");
 
-table.id = "table";
-table.classList.add("table");
+const createAttributedElements = (tag, attrObj, str) => {
```

Comment:
```suggestion
const createAttributedElements = (tag, attrObj, str = "") => {
```
こうすると、その後のif分が不要になりそうです

Reply:
なるほどです！
初期値を入れることについては思いつきませんでした。ご教授ありがとうございます。
修正しました。
[f1d3e95](https://github.com/443n0511/js-lesson/pull/44/commits/f1d3e95163989f60fb3379ad701544f9aeb5a316)